plugins/techdocs/src/reader/components/TechDocsReaderPageContent/TechDocsReaderPageContent.tsx

```diff
@@ -78,8 +80,18 @@ export const TechDocsReaderPageContent = withTechDocsReaderProvider(
       entityRef,
       setShadowRoot,
     } = useTechDocsReaderPage();
-
     const dom = useTechDocsReaderDom(entityRef);
+    const hash = window.location.hash;
+    const isStyleLoading = useShadowDomStylesLoading(dom);
+    const [hashElement] = useShadowRootElements([`[id="${hash.slice(1)}"]`]);
+
+    useEffect(() => {
+      if (hashElement && !isStyleLoading) {
+        hashElement.scrollIntoView();
+      } else {
+        document?.querySelector('header')?.scrollIntoView();
```

Comment:
Scrolling the Backstage `header` element into view effectively scrolls user to the top of the page.

Reply:
Is this because it always takes the first one?

Reply:
Yes it always takes the first one. I was hesitant to do a more specific selection because the header component is different internally vs open source Backstage.